src/porepy/models/verification_setups/terzaghi.py

```diff
@@ -0,0 +1,723 @@
+"""
```

Comment:
I think verification_setups should be moved from models to applications.

Reply:
I agree. I will move them.

Reply:
Fair enough.